### PR TITLE
Update Readme.md to enable login credentials

### PR DESCRIPTION
**Pull request by sakshi1499**
Fixes #

#### Describe the changes you have made in this pr -
This PR fixes issue #518 .

### CODE REVIEW

**Comment by sakshi1499** [comment]
Do you want me to add this in the readme itself? Or the link to the file or wiki? I feel adding the whole thing in the readme will make it too long unnecessarily. Maybe a link would seem cleaner. What do you suggest?

**Comment by tachyons** [comment]
just mention run rake db:seed in development environment to populate sample data. No need to mention what all data are populated

**Comment by sakshi1499** [comment]
Okay, will update it that way :)

**Comment by sakshi1499** [comment]
I was not getting what you wrote. I think what you mean to convey is that this command rake db:seed not only seeds the admin account but also other things. So instead of explicitly mentioning admin account I should mention that it populates the sample data and loads it, right? Correct me if I am wrong

**Comment by tachyons** [comment]
Yes

**Comment by sakshi1499** [comment]
@tachyons check if it is better now?

File: README.md [hunk 1]
```diff
@@ -65,7 +65,9 @@ docker-compose build --no-cache
 ```
 
 ## Developer Instructions
-Developers can quickly get started by setting up the dev environment using the instructions above. The database is seeded with the following admin account.
+Developers can quickly get started by setting up the dev environment using the instructions above. It is necessary to execute 
+'bundle exec rake db:seed' to populate the sample data and access admin account.
```
Comment (ankitkataria) [inline README.md]: It's not necessary to seed the db for starting development, how does the following look? 

> To seed the database with some sample data, run `bundle exec rake db:seed`. The admin credentials after seeding will be as follows ...

Comment (sakshi1499) [reply README.md]: It seems good. I'll make these changes :)

Comment (sakshi1499) [reply README.md]: I think you already made the changes. Right? @ankitkataria

**Comment by sakshi1499** [comment]
@ankitkataria This has to be closed too right?

**Pull request merged**